yt/frontends/gadget/data_structures.py

```diff
@@ -561,16 +561,46 @@ def _is_valid(cls, filename, *args, **kwargs):
 
 
 class GadgetHDF5File(ParticleFile):
+
+    _fields_with_cols = ["Metallicity_", "PassiveScalars_", "Chemistry_", "GFM_Metals_"]
```

Comment:
```suggestion
    _fields_with_cols = ("Metallicity_", "PassiveScalars_", "Chemistry_", "GFM_Metals_")
```
What about a variable name that indicates they're prefixes?  I can't think of one that's succinct, so maybe no...

Reply:
ya, I was struggling with the name of this `_fields_with_cols` variable. `_prefixed_fields` could work, but doesn't capture how the column is set by the final integer string. Will definitely change it to a tuple instead of a list and think about the name more.